openpype/hosts/houdini/plugins/load/load_camera.py

```diff
@@ -90,13 +97,8 @@ class CameraLoader(load.LoaderPlugin):
 
     def load(self, context, name=None, namespace=None, data=None):
 
-        import os
-        import hou
-
         # Format file name, Houdini only wants forward slashes
-        file_path = self.filepath_from_context(context)
-        file_path = os.path.normpath(file_path)
-        file_path = file_path.replace("\\", "/")
+        file_path = self.fname.replace("\\", "/")
```

Comment:
Didn't we deprecate `self.fname` some time ago @iLLiCiTiT ?

Reply:
I found it's still defined

Reply:
Found it. It's this PR which tried to eradicate it as a first step: https://github.com/ynput/OpenPype/pull/4602